qutebrowser/browser/hints.py

```diff
@@ -236,6 +236,7 @@ def yank(self, url, context):
                utils.supports_selection())
 
         urlstr = url.toString(QUrl.FullyEncoded | QUrl.RemovePassword)
+        urlstr = urlstr.lstrip('mailto:')
```

Comment:
That doesn't do the right thing - it strips any of the characters m, a, i, l, t, o and :.

It'd probably be cleaner to do this before converting the URL to a string.